lib/libutils/ext/trace.c

```diff
@@ -53,12 +53,13 @@ int trace_get_level(void)
 
 static const char *trace_level_to_string(int level, bool level_ok)
 {
-	static const char lvl_strs[][4] = {
-		"UKN", "ERR", "INF", "DBG", "FLW" };
+	static const char lvl_strs[][9] = {
+		"UNKNOWN:", "ERROR:  ", "INFO:   ", "DEBUG:  ",
```

Comment:
Why do we need these long names? I'd rather have "U", "E", "I", "D", "F" instead.